src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -334,7 +334,7 @@
           "How many times to maximum attempt to flush SATB buffers at the " \
           "end of concurrent marking.")                                     \
                                                                             \
-  product(bool, ShenandoahSuspendibleWorkers, false, EXPERIMENTAL,          \
+  product(bool, ShenandoahSuspendibleWorkers, true, EXPERIMENTAL,           \
```

Comment:
Given that turning this off would most likely result in correctness issues, or certainly will in the future when Lilliput arrives, we should consider removing this option altogether.

Reply:
I agree that removing the flag is a good idea. Anyone recall why it was put there in the first place?

Reply:
Separate PR would be preferred. At that time soak-test with value "off" to peek inside the pandora's box (which might be empty for all we know). Perhaps repeat the same experiment with Lilliput to see if it reveals the issues that @rkennke alludes to?